client/me/security-checkup/recovery-email.jsx

```diff
@@ -59,25 +59,29 @@ module.exports = React.createClass( {
 						/>
 				</ManageContact>
 		);
-	},
+	}
 
-	haveEmail: function() {
+	haveEmail() {
```

Comment:
Fat arrow is generally only necessary for functions used as callbacks (since they lose the context of `this`), so I went to trace where its used... but couldn't find any references. It looks like this function is dead code?